src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -1199,4 +1205,13 @@ public void SetEnvironmentId(Guid environmentId)
             _environmentId = environmentId;
         }
     }
+
+    public enum EnvironmentType
```

Comment:
This shouldn't be in Voron, this is a RavenDB concept, not a Voron concept

Reply:
Hmm, maybe it would be better, instead of adding all of those fields to Voron, add 'OnBeforeDirectoryCreated' event and attach to it in RavenDB?

Reply:
Yes, absolutely.